vue-components/.browserslistrc

```diff
@@ -2,8 +2,7 @@ last 2 Chrome versions
 last 2 Firefox versions
 last 2 Opera versions
 last 2 Edge versions
-IE >=11
-Safari >=5.1
-iOS >=6.1
-Android >=4.1
-defaults
+
```

Comment:
Intentional blank line?

Reply:
Yes :/. But I rather not add another commit, the browser tests are green for once. That will be fixed once wikimedia/browserslist-config-wikimedia#6 is merged and this file is replaced by just referencing the modern-es6 config.